src/keys/KeyManager.ts

```diff
@@ -38,28 +37,26 @@ class KeyManager {
   public readonly rootCertPath: string;
   public readonly rootCertsPath: string;
   public readonly dbKeyPath: string;
-  public readonly vaultKeyPath: string;
+  public readonly litmusMessagePath: string;
```

Comment:
I'm pretty sure this was meant to be called the `canary`. It can just be `canaryPath`.

It was meant to tell us if our descryption is successful by first encrypting a known static value, and then descrypting to see if it is the known static value.

It should have it's own custom exception if it failed in the `keys/errors.ts` and then trigger failure on `pk bootstrap` and `pk agent start`.

Reply:
From what I can tell this shouldn't exist anymore. might of snuck in when the vault key was removed. I'll remove it.